tests/cpython-tests/run-tests.sh

```diff
@@ -1,25 +1,26 @@
 #!/bin/bash
 TEST_FILE=tests.all
+if [[ ("$#" -eq 1) && (-f $1) ]]; then
```

Comment:
This file is called `run_tests_locally.sh` in other test suites.

Reply:
Renamed.